### ISSUE TITLE
Explorer can't display data due to CORS errors

### ISSUE THREAD

**Issue by frei-geist** [symbol/symbol-explorer#974]
Hello. 
Latest explorer throws the following error in the console
`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://symbol.services/nodes?filter=suggested&limit=30&ssl=true. (Reason: CORS request did not succeed).`

I have tested this in firefox  94.0 64-bit for ubuntu and the 
Chromium web browser version 96.0. also on ubuntu.

BTW: add blockers were disabled.

The same problem happens in the latest desktop universal wallet.

I'm attaching full console output from both browsers in the zip file.
[errors.zip](https://github.com/symbol/symbol-explorer/files/7576400/errors.zip)

The outputs were generated while visiting https://symbol.fyi page.

**Comment by AnthonyLaw** [comment]
Hi there, thanks for the feedback.
are you able visit `symbol.services/nodes?filter=suggested&limit=30&ssl=true` this url on your machine?

**Comment by frei-geist** [comment]
Hello. No i can't access this site from browser.
I have also tried in console with wget and the following message is given:
`user@machine:~$ --2021-11-22 21:26:51--  https://symbol.services/nodes?filter=suggested
Resolving symbol.services (symbol.services)... failed: Temporary failure in name resolution.
wget: unable to resolve host address ‘symbol.services’`

It seems a problem with local router firewall or ISP.
I tried to access the URL from another network and it works.

**Comment by AnthonyLaw** [comment]
> It seems a problem with local router firewall or ISP.

I'm not sure why the firewall is blocking the URL.
maybe it needs to whitelist the URL on your router?

**Comment by frei-geist** [comment]
Hello. It seems is not a router I haven't checked it but something
strange happened that makes me doubt that is router problem.

I wanted to check the site 
http://forkwatch.symbol.tools/nem/summary
and I' ve got the same problem. I know 2 days ago I was able to access the site normally and
today not any more. 
I tried to ping the domain but was getting 
ping: unknown host forkwatch.symbol.tools

Then I went here
https://domaintoipconverter.com/index.php
to find out the IP address of the site.
Then I have put the IP  address and the domain into the `/etc/hosts` file and retried.
The site loaded. I have then make the same for the https://symbol.services and the 
explorer worked.

I think is something with the DNS that the ISP provider uses.
